certora/README.md

```diff
@@ -19,18 +21,26 @@ Please ensure that `CERTORAKEY` is set up in your environment.
 The Bundler3 contract enables an EOA to call different endpoint contracts onchain as well as grouping several calls in a single bundle.
 These calls may themselves reenter the Bundler3 contract.
 
-### Folders and file structure
+## Bundler3 safety restrictions
 
-The [`certora/specs`](specs) folder contains these files:
+A key feature of the bundler is to restrict reentering in adapter calls.
+That is we check that reentering an adapter is only possible during a multicall execution.
+We also check that adapters' methods used during a bundle execution may only be called by the Bundler3 contract.
+We check that it's only possible to reenter the adapter using a dedicated Bunlder3 function.
+Bundler3 uses transient storage, it's checked that it's nullified on each entry-point call.
+This is checked with a separate configuration as it requires to disable sanity-checks (because `reenter` cannot be an entry-point _per se_).
 
-- [`Bundler3.spec`](specs/Bundler3.spec) checks Bundler3 entry points behave as expected;
-- [`AllowancesInvariant.spec`](specs/AllowancesInvariant.spec) check that token allowances are reset;
-- [`GeneralAdapter1Reverts.spec`](specs/GeneralAdapter1Reverts.spec) checks that state changes when adapter functions are called, or the execution reverts;
-- [`MorphoZeroConditions.spec`](specs/MorphoZeroConditions.spec) checks that calls to Morpho with zero inputs that revert in Morpho make the adapter revert;
```

Comment:
I think the zero condition is missing from the description, but maybe not necessary to add (as it's a bit technical)